manifests/capacityscheduling/capacityscheduling_v1alpha1_elasticquota.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
CRD has been GAed, so use `v1`?

And rename the file as "crd.yaml" should be good enough.

Reply:
Done